qutip/solver/mcsolve.py

```diff
@@ -148,8 +148,7 @@ def mcsolve(H, state, tlist, c_ops=(), e_ops=None, ntraj=500, *,
 
     if isinstance(ntraj, list):
```

Comment:
I would change this to:
```
if isinstance(ntraj, (list, tuple)):
    raise TypeError(
        "ntraj must be an integer. A list of numbers is not longer supported."
    )
```
This covers both lists and tuples and explains what ntraj should be.

Apologies for not making this a suggestion -- GitHub seems not to allow suggestions if there are deleted lines, which is a bit odd.